packages/frontend/core/src/blocksuite/block-suite-editor/blocksuite-editor.tsx

```diff
@@ -317,22 +317,16 @@ export const BlockSuiteEditor = (props: EditorProps) => {
   useEffect(() => {
     if (props.page.root) {
       setIsLoading(false);
+      track.doc.$.$.loadDoc({
```

Comment:
https://github.com/toeverything/AFFiNE/blob/release/packages/frontend/core/src/desktop/pages/workspace/share/share-page.tsx#L150

Reply:
The tracking points added here are also effective in the share page.

Reply:
I mean use the same logic to track doc load